scripts/github_api

```diff
@@ -59,19 +60,34 @@ def _request(auth, method, url, **kwargs):
     return response
 
 
+def _print_output(output):
+    """
+    Prints output from this script in a key=value format.
+
+    Args:
+        output (dict): values to be formatted
+    """
+    for k, v in output.items():
+        print '%s="%s"' % (k, v)
+
+
 class PullRequest:
```

Comment:
not related exactly to this patch: It seems to me this class is not a PullRequest, it is a PullRequestManager, as it creates (open() method) PRs and queries them (get() method). I'd prefer to name this better to avoid confusion.